lib/cog/commands/rule/drop.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Cog.Commands.Rule.Drop do
+  @moduledoc """
+  Drops rules by id or all rules for a specific command.
+
+  USAGE
+    rule drop [FLAGS] [OPTIONS] [<id> ...] 
+
+  FLAGS
+    -h, --help  Display this usage info
+
+  OPTIONS
+    -c, --command  Drops rules belonging to the command
+
+  ARGS
```

Comment:
Not that it matters, but I've been putting `ARGS` at the top. My thinking was that `ARGS` are more important since they're often required.